x-pack/test/functional/apps/ingest_pipelines/ingest_pipelines.ts

```diff
@@ -7,9 +7,32 @@
 import expect from '@kbn/expect';
 import { FtrProviderContext } from '../../ftr_provider_context';
 
+const PIPELINE = {
+  name: 'test_pipeline',
+  description: 'My pipeline description.',
+  version: 1,
+  processors: JSON.stringify([
+    {
+      set: {
+        field: 'foo',
+        value: 'new',
+      },
+    },
+  ]),
+  onFailureProcessors: JSON.stringify([
+    {
+      set: {
+        field: '_index',
+        value: 'failed-{{ _index }}',
+      },
+    },
+  ]),
+};
+
 export default ({ getPageObjects, getService }: FtrProviderContext) => {
   const pageObjects = getPageObjects(['common', 'ingestPipelines']);
   const log = getService('log');
+  const es = getService('legacyEs');
 
   describe('Ingest Pipelines', function() {
     this.tags('smoke');
```

Comment:
I'm not too familiar with this tag. @cuff-links do you know if it should still remain now that I've added a second test?

Reply:
This tag means that the test runs on both chrome and firefox. I think that it should be good to stay.